### PR TITLE
add test-cases for longitude wrapping

### PR DESCRIPTION
**Pull request by sevko**
See the related pelias/api#121 pull request.

fixes pelias/api#56

### CODE REVIEW

**Comment by dianashk** [comment]
:+1:

**Comment by orangejulius** [comment]
Good tests, one thing that's not good though is that the failure mode for these tests (like if say, you run these tests against an API codebase that does NOT support wrapping) is the 400 returned by the API causing the test suite to quit immediately. But there's not a straightforward way in our current test framework to catch the 400. Could be done though.

**Comment by orangejulius** [comment]
With pelias/fuzzy-tester#10 merged, this would be less of a problem :)

File: locations.json [hunk 1]
```diff
@@ -30,5 +30,15 @@
     "admin0": "United States",
     "admin1": "Massachusetts",
     "admin1_abbr": "MA"
+  },
```
Comment (dianashk) [inline locations.json]: this doesn't seem related to the lat/lon wrapping fix

**Comment by dianashk** [comment]
This needs a few tests for `lat` wrapping now.

File: locations.json (hunk 1 above)
Comment (orangejulius) [inline locations.json]: This is fixed since locations.json went away in the meantime. Github may or may not show it as changed, but it's not changed in the final version of this branch.

**Comment by orangejulius** [comment]
Tests added!

**Comment by dianashk** [comment]
:+1:
